OKTA- 1016057 - Terraform OIG Guide (#5828)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/terraform-oig-resources/main/campaigns/overview.md

> Okta Access Certification Campaigns is a core Okta Identity Governance feature that allows stakeholders to conduct scheduled reviews of user access to apps and resources. Based on the reviews, Okta can then modify resource access so that the users are provided with only have the access they need. See [Campaigns](https://help.okta.com/okta_help.htm?type=oie&id=campaigns.htm) and [Campaigns API](https://developer.okta.com/docs/api/iga/openapi/governance.api/tag/Campaigns/).

Reviewer: ```suggestion
Okta Access Certification Campaigns is a core Okta Identity Governance feature that allows stakeholders to conduct scheduled reviews of user access to resources, such as apps and groups. See [Campaigns](https://help.okta.com/okta_help.htm?type=oie&id=<NEED campaign alias>) in the product documentation and the [Campaigns](https://developer.okta.com/docs/api/iga/openapi/governance.api/tag/Campaigns/) API.
```
